ploneorg/theme/theme/plone-open-garden.html

```diff
@@ -2397,7 +2358,7 @@ <h2>How do I get there?</h2>
            </div>
            <div id="here-tab3" class="tab-pane">
              <p>From Naples Central Station take the local “Circumvesuviana” train for Sorrento and get off at the Sant
-                 Agnello stop. From Sant Agnello station we advise you to take a taxi or a walk (of about 10 minutes),
+                 Agnello stop. From Sant Agnello station you can take a taxi or walk (about 10 minutes),
```

Comment:
This 10 minutes figure kills me...it has taken me way longer than that, always a source of amusement for others

Reply:
Yes you're right! It is also said to be "a few minutes’ walk from the city center" under "Being guests at the Mediterraneo" section - as if it takes a few minutes to walk to the center of Sorrento! Anyway - I'll up it to 15 minutes here...

Reply:
Changed 10 to 15 minutes.